keps/sig-node/2000-graceful-node-shutdown/kep.yaml

```diff
@@ -6,18 +6,26 @@ authors:
 owning-sig: sig-node
 status: implementable
 creation-date: 2020-09-21
+reviewers:
+  - TBD
```

Comment:
Looks like Sergey and Karan were the original reviewers?

Reply:
Yup, added thanks!